app/assets/stylesheets/application.scss

```diff
@@ -59,6 +59,10 @@ nav {
   margin-bottom: 25px;
 }
 
+.item a {
+  color: #666;
```

Comment:
Color literals like `#666` should only be used in variable declarations; they should be referred to via variable everywhere else.